POLYFILLS.md

```diff
@@ -6,20 +6,61 @@ browsers yet. In order to maximize browser compatibility, you should install
+WebXR Device API          |     🚫 |     🚧 |        🚫 |         🚫 |   🚫 |   🚫
+WebXR HitTest API         |     🚫 |     🚧 |        🚫 |         🚫 |   🚫 |   🚫
+
+
+_1: Resize Observer is optional. [See below](https://github.com/PolymerLabs/model-viewer/blob/master/POLYFILLS.md#regarding-resize-observer) for more details_
+
+_2: Intersection Observer is optional. [See below](https://github.com/PolymerLabs/model-viewer/blob/master/POLYFILLS.md#regarding-intersection-observer) for more details_
+
+_3: Fullscreen API is only needed when using unstable WebXR features. [See below](https://github.com/PolymerLabs/model-viewer/blob/master/POLYFILLS.md#regarding-fullscreen-api) for more details_
+
+### Regarding IE 11
+
+We currently test and support Internet Explorer 11. We also distribute a special
+["legacy" bundle](https://unpkg.com/@google/model-viewer/dist/model-viewer-legacy.js)
+that is compatible with IE 11 but comes with the following important caveat:
+the "legacy" bundle includes JavaScript language feature polyfills and code
+transformations that will incur a byte size and execution time penalty
+compared to the non-legacy versions of the code.
+
 ## Browser Support
 
-The following emerging standard web platform APIs are required by
+The following emerging standard web platform APIs are *required* by
 `<model-viewer>`:
 
  - [Custom Elements](https://html.spec.whatwg.org/multipage/custom-elements.html#custom-elements) ([CanIUse](https://caniuse.com/#feat=custom-elementsv1), [Platform Status](https://www.chromestatus.com/features/4696261944934400))
  - [Shadow DOM](https://dom.spec.whatwg.org/#shadow-trees) ([CanIUse](https://caniuse.com/#feat=shadowdomv1), [Platform Status](https://www.chromestatus.com/features/4667415417847808))
- - [Intersection Observer](https://w3c.github.io/IntersectionObserver/) ([CanIUse](https://caniuse.com/#feat=intersectionobserver), [Platform Status](https://www.chromestatus.com/features/5695342691483648))
- - [Fullscreen API](https://fullscreen.spec.whatwg.org/) ([CanIUse](https://caniuse.com/#feat=fullscreen), [Platform Status](https://www.chromestatus.com/features/6596356319739904))
 
-The following emerging web platform API is optional, and will be used by
-`<model-viewer>` if it is detected on the page:
+The following emerging web platform APIs are *optional*, and will be used by
+`<model-viewer>` if they are detected on the page:
 
  - [Resize Observer](https://wicg.github.io/ResizeObserver/) ([CanIUse](https://caniuse.com/#feat=resizeobserver), [Platform Status](https://www.chromestatus.com/features/5705346022637568))
+ - [Intersection Observer](https://w3c.github.io/IntersectionObserver/) ([CanIUse](https://caniuse.com/#feat=intersectionobserver), [Platform Status](https://www.chromestatus.com/features/5695342691483648))
+ - [Fullscreen API](https://fullscreen.spec.whatwg.org/) ([CanIUse](https://caniuse.com/#feat=fullscreen), [Platform Status](https://www.chromestatus.com/features/6596356319739904))
```

Comment:
```suggestion
 - [Fullscreen API](https://fullscreen.spec.whatwg.org/) ([CanIUse](https://caniuse.com/#feat=fullscreen), [Chrome Platform Status](https://www.chromestatus.com/features/6596356319739904))
```

Reply:
Would you like to make this change for all links previously titled "Platform Status"?